### ISSUE TITLE
deno.enablePaths does not ignore deno.enable

### ISSUE THREAD

**Issue by Reububble** [denoland/deno#20411]
I just updated to Deno 1.36.4 from 1.34 and VSCode 1.82 from 1.81 and now I'm getting no intellisense in my vscode project. Deno Language Server says `Error converting specifier settings: invalid type: null, expected a boolean`. I figured that it might be referring to `deno.enable`, which I don't specify since I have been using `deno.enablePaths`. Setting `deno.enable` (either true or false) resolves the intellisense issue. Removing it does not bring back the issue until reloading the window. 

VSCode info:
> Version: 1.82.0 (user setup)
> Commit: 8b617bd08fd9e3fc94d14adb8d358b56e3f72314
> Date: 2023-09-06T22:07:07.438Z
> Electron: 25.8.0
> ElectronBuildId: `23503258`
> Chromium: 114.0.5735.289
> Node.js: 18.15.0
> V8: 11.4.183.29-electron.0
> OS: Windows_NT x64 10.0.18362

Deno extension: v3.21.0

I'm able to reproduce the issue with 2 files in a folder.

bug.code-workspace:
```
{
  "folders": [
    {
      "path": "."
    }
  ],
  "settings": {
    // "deno.enable": false, // Uncomment to fix intellisense
    "deno.enablePaths": [ "." ]
  }
}
```

bug.ts:
```
Deno.haltAndCatchFire(); // This should be an error
```

**Comment by bartlomieju** [comment]
CC @nayeemrmn

**Comment by nayeemrmn** [comment]
This is a compat issue that will be fixed with Deno 1.37. Basically https://github.com/denoland/vscode_deno/pull/902 adds a compat layer which turns `"enable": null` to `"enable": false`, but that only applies to creating the initialization options, not the `workspace/configuration` requests that VSCode handles directly in the background. I'm not sure we can do anything about it.

Essentially we've already landed the fix here, it's just the extension change wasn't FULLY backward-compatible though it avoids crashes on init.

**Comment by bartlomieju** [comment]
@nayeemrmn could you hot fix the extension in the mean time?
